app/javascript/src/Navigation/components/header/DocumentationMenu.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import DocumentationMenuItem from 'Navigation/components/header/DocumentationMenuItem'
+
+let menuItemKey = 0
+const renderMenuItems = (docsLinksClass, items) => items.map(
+  item => <DocumentationMenuItem key={`docsMenuItem${menuItemKey++}`} docsLinksClass={docsLinksClass} item={item}/>
```

Comment:
```js
items.map(
  (item, index) => <DocumentationMenuItem key={`docsMenuItem${index}`} docsLinksClass={docsLinksClass} item={item}/>
```
that way you don't need to declare `menuIitemKey`

Reply:
Using items index as keys [are discouraged](https://reactjs.org/docs/lists-and-keys.html#keys), better to use some other unique property like name or something.